CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 5.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Fix compare methods between DateTime(0) and None.
```

Comment:
```suggestion
- Fix compare methods between DateTime instances and None.
```

Reply:
As `DateTime(0)` is the only problematic case, I think that mentioning it explicitly is more descriptive for this change.

Reply:
But people who look at the change log won't know what that `DateTime(0)` means. You improve peoples' understanding of the change by keeping the message more generic.

Reply:
OK, done.